### PR TITLE
Use BGP routing table for asnlookup expert

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: contrib/cron-jobs/qualle.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+# # m h  dom mon dow   command
+# 02  01 *   *   *     ( cd /tmp; /my/path/to/this/script )
+#
+
+cd /tmp
+mkdir qualle
+cd qualle
+
+echo "Download IPv4 table from 'qualle.cert.at'."
+whois -h qualle.cert.at block origin v4table > ipasn.dat
```
Comment (aaronkaplan) [inline contrib/cron-jobs/qualle.sh]: actually that's not really public :) But ok. No big secret either. Anyway... maybe let's abstract this by using a variable. Like:

```bash
BGPROUTER=....

...
whois -h $BGPROUTER block origin v4table > ipasn.dat
```

File: contrib/cron-jobs/qualle.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+fi
+
+python3 -c "import pyasn; pyasn.pyasn('/tmp/qualle/ipasn.dat')" > /dev/null
+if [[ $? -ne 0 ]]; then
+    echo "Unparseable data."
+    exit 1
+fi
+
+echo "Applying new data to '/opt/intelmq/var/lib/bots/asn_lookup/'."
+mv ipasn.dat  /opt/intelmq/var/lib/bots/asn_lookup/
```
Comment (aaronkaplan) [inline contrib/cron-jobs/qualle.sh]: Any other sanity checks needed before moving it there?

Does the bot need to be HUPed?

**Review by aaronkaplan — CHANGES_REQUESTED** [review]

File: contrib/cron-jobs/qualle.sh (hunk 2 above)
Comment (ghost) [reply contrib/cron-jobs/qualle.sh]: > Any other sanity checks needed before moving it there?

I can add more if you have any ideas.
> Does the bot need to be HUPed?

The bot id depends on the setup and there may be multiple of them.

File: contrib/cron-jobs/qualle.sh (hunk 1 above)
Comment (ghost) [reply contrib/cron-jobs/qualle.sh]: I renamed the script to make it more generic.

qualle.cert.at is resolveable and pingable from the web.

**Comment by aaronkaplan** [comment]
looks ok now, thx! May I merge?

**Comment by ghost** [comment]
> looks ok now, thx! May I merge?

Sure

**Pull request merged**